src/pages/games/witchcraft.astro

```diff
@@ -0,0 +1,378 @@
+---
+import { Markdown } from 'astro/components'
+import BasicMeta from '../../components/BasicMeta.astro'
+import Logo from '../../components/logo/Witchcraft.astro'
+import Noise from '../../components/Noise.vue'
+---
+<html lang="en">
+	<head>
+		<BasicMeta/>
+		<title>Witchcraft</title>
+		<meta name="theme-color" content="#e9e1f9">
+		<style>
+			html
+			{
+				font-family: 'Times New Roman', Times, serif;
+				font-size: 28px;
+				line-height: 1.4;
+				background: rgb(122, 122, 122);
+			}
+
+			:global(::selection)
+			{
+				background-color: #c40001;
+			}
+
+			#overlay
+			{
+				position: fixed;
+				top: 0;
+				left: 0;
+				right: 0;
+				bottom: 0;
+				pointer-events: none;
+				z-index: 999;
+
+				opacity: 0.7;
+				box-shadow: inset 0 0 100px 50px #000;
+				background: radial-gradient(hsla(0, 0%, 0%, 0) 0%,
+					hsla(0, 0%, 0%, 0.013) 10.6%,
+					hsla(0, 0%, 0%, 0.049) 20.3%,
+					hsla(0, 0%, 0%, 0.104) 29.2%,
+					hsla(0, 0%, 0%, 0.175) 37.4%,
+					hsla(0, 0%, 0%, 0.259) 45%,
+					hsla(0, 0%, 0%, 0.352) 52%,
+					hsla(0, 0%, 0%, 0.45) 58.5%,
+					hsla(0, 0%, 0%, 0.55) 64.6%,
+					hsla(0, 0%, 0%, 0.648) 70.3%,
+					hsla(0, 0%, 0%, 0.741) 75.6%,
+					hsla(0, 0%, 0%, 0.825) 80.7%,
+					hsla(0, 0%, 0%, 0.896) 85.7%,
+					hsla(0, 0%, 0%, 0.951) 90.5%,
+					hsla(0, 0%, 0%, 0.987) 95.2%,
+					hsl(0, 0%, 0%) 100%);
+			}
+
+			main
+			{
+				max-width: 70ch;
+				padding: 0 20px;
+				margin: 100px auto;
+			}
+
+			:global(a)
+			{
+				color: #eee
+			}
+
+			:global(h2)
```

Comment:
Unfortunately, we need to wrap a lot of these CSS properties in the `:global()` function Astro provides so that they can target the HTML contained inside the `<Markdown>` component.

Reply:
There is prob a deep selector

Reply:
This is the deep selector.